compass/landice/tests/humboldt/restart_test/streams.restart

```diff
@@ -4,7 +4,7 @@
                   filename_template="rst.$Y_$M_$D.nc"
                   filename_interval="output_interval"
                   reference_time="2007-01-01_00:00:00"
-                  output_interval="0001-00-00_00:00:00"/>
+                  output_interval="0000-02-00_00:00:00"/>
```

Comment:
Does this 2-month output interval actually work with the run durations defined in namelist.restart.hydro and namelist.restart.rst.hydro?

Reply:
Yes, because we only need a restart file at 2 months in (the run duration of the first run).  The output stream is writing output every day, so we get all the output we need (too much, really).